### PR TITLE
Uml 178 field validation

### PR DESCRIPTION
**Pull request by taylorsj1980**

### CODE REVIEW

File: service-front/app/src/Actor/src/Handler/CreateAccountHandler.php [hunk 1]
```diff
@@ -25,6 +24,8 @@
  */
 class CreateAccountHandler extends AbstractHandler
 {
+    use CsrfGuard;
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Handler/CreateAccountHandler.php]: don't forget to add the CsrfGuardAware interface.

File: service-front/app/src/Viewer/src/Handler/EnterCodeHandler.php [hunk 2]
```diff
@@ -19,6 +18,7 @@
  */
 class EnterCodeHandler extends AbstractHandler
 {
+    use CsrfGuard;
```
Comment (cooperaj) [inline service-front/app/src/Viewer/src/Handler/EnterCodeHandler.php]: interface

File: service-front/app/src/Actor/src/Handler/CreateAccountHandler.php (hunk 1 above)
Comment (taylorsj1980) [reply service-front/app/src/Actor/src/Handler/CreateAccountHandler.php]: Good spot. Will add. Thanks

File: service-front/app/src/Viewer/src/Handler/EnterCodeHandler.php (hunk 2 above)
Comment (taylorsj1980) [reply service-front/app/src/Viewer/src/Handler/EnterCodeHandler.php]: Good spot. Will add. Thanks

**Pull request merged**